packages/babel-preset-react-app/create.js

```diff
@@ -90,8 +90,8 @@ module.exports = function(api, opts, env) {
           // If users import all core-js they're probably not concerned with
           // bundle size. We shouldn't rely on magic to try and shrink it.
           useBuiltIns: false,
-          // Do not transform modules to CJS
-          modules: false,
+          // Setting this to false will not transform modules.
+          modules: useESModules ? false : 'cjs',
```

Comment:
This isn't the same behavior -- we never want to compile modules unless the user explicitly specifies `useESModules` as false.

Maybe piggybacking off this flag isn't the best idea -- is there any prior art on how this should be handled? Should we just add support for the `modules` flag?

Reply:
Another option is to set `useESModules` to `true` by default (regardless of env) and rewrite this https://github.com/facebook/create-react-app/blob/aaf7186627e68ed33353d8c51c94985941277216/packages/babel-preset-react-app/create.js#L161-L164
as `useESModules: useESModules && !isEnvTest`. Will this match the initial behavior?

-----
Having two different options for the seemingly the same can be confusing.

Reply:
My use case might be special, but I do not want to always use `modules: 'cjs'` when using `useESModules: false`. Omitting it in the initial PR was a conscious decision so adding a separate option would be nice.

Reply:
I added new "modules" option